src/components/Create/components/Selection/SelectionCard.tsx

```diff
@@ -1,51 +1,36 @@
 import { Divider } from 'antd'
-import * as styleColors from 'constants/styles/colors'
-import { ThemeContext } from 'contexts/themeContext'
 import { ReactNode, useCallback, useContext, useMemo } from 'react'
+import { classNames } from 'utils/classNames'
 import { CheckedCircle, RadialBackgroundIcon } from './components'
 import { SelectionContext } from './Selection'
 
-const Container: React.FC<{ isSelected: boolean; isDefocused: boolean }> = ({
-  isDefocused,
-  isSelected,
-  children,
-}) => {
-  const {
-    isDarkMode,
-    theme: { colors },
-  } = useContext(ThemeContext)
-
-  const borderColor = useMemo(() => {
-    if (isSelected) return colors.stroke.action.primary
-    if (isDefocused) {
-      return isDarkMode
-        ? styleColors.darkColors.darkGray500
-        : styleColors.lightColors.warmGray200
-    }
-    return isDarkMode
-      ? styleColors.darkColors.darkGray300
-      : styleColors.lightColors.warmGray300
-  }, [colors.stroke.action.primary, isDarkMode, isDefocused, isSelected])
-
-  const backgroundColor = useMemo(() => {
-    if (isDefocused) {
-      return isDarkMode
-        ? styleColors.darkColors.darkGray800
-        : styleColors.lightColors.juiceLightest
-    }
-    return isDarkMode ? styleColors.darkColors.darkGray600 : undefined
-  }, [isDarkMode, isDefocused])
+const Container: React.FC<{
+  isSelected: boolean
+  isDefocused: boolean
+  isDisabled: boolean
+}> = ({ isDefocused, isSelected, isDisabled, children }) => {
+  const borderColorClassNames = useMemo(() => {
+    if (isSelected) return 'border-haze-500'
+    return classNames(
+      !isDisabled ? 'hover:border-smoke-500 dark:hover:border-slate-100' : '',
+      isDefocused
+        ? 'border-smoke-200 dark:border-slate-500'
+        : 'border-smoke-300 dark:border-slate-300',
+    )
+  }, [isDefocused, isDisabled, isSelected])
```

Comment:
Not stoked that we're using JS for these states. It would be a lot better on all levels to use css pseudo classes https://tailwindcss.com/docs/hover-focus-and-other-states#disabled

Reply:
agreed. Lets make an effort to do this after though I reckon